### ISSUE TITLE
AttributeError: 'Twarc2' object has no attribute 'retweeed_by'

### ISSUE THREAD

**Issue by miguelwon** [DocNow/twarc#620]
Hi,
I'm having an error when using `retweeed_by`. It's a fresh twarc installation and I'm using the example given in the [docs](https://github.com/DocNow/twarc/blob/main/docs/twarc2_en_us.md#retweets).

```
(env3.8) mwon@mwon:/mwon_disk/disk2/paperChega/replicate/results/retweets_lideres$ twarc2 version
twarc v2.10.1
(env3.8) mwon@mwon:/mwon_disk/disk2/paperChega/replicate/results/retweets_lideres$ twarc2 retweeted-by 1460417326130421765 retweeting-users.jsonl
Traceback (most recent call last):
  File "/mwon_disk/disk2/paperChega/env3.8/bin/twarc2", line 8, in <module>
    sys.exit(twarc2())
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/click/decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/twarc/decorators2.py", line 180, in __call__
    return self.f(*args, **kwargs)
  File "/mwon_disk/disk2/paperChega/env3.8/lib/python3.8/site-packages/twarc/command2.py", line 970, in retweeted_by
    func=T.retweeed_by,
AttributeError: 'Twarc2' object has no attribute 'retweeed_by'
```

**Comment by SamHames** [comment]
Thanks for reporting, the next release will have that fixes.

**Comment by edsu** [comment]
@miguelwon A fix for this was just released in v2.10.2 -- thanks for reporting it!
